style.css

```diff
@@ -0,0 +1,598 @@
+  position: relative;
+  margin-bottom: 1.2rem;
+}
+
+.speakers-list-item__role::after {
+  content: " ";
+  position: absolute;
+  bottom: -0.8rem;
+  left: 0;
+  background: #383b42;
+  width: 1rem;
+  height: 0.1rem;
+}
+
+.speakers-list-item__text {
+  font-size: 0.8rem;
+}
+
+/* THE PAST EVENT SECTION IN THE ABOUT US PAGE */
+
+.summits {
+  text-align: center;
+}
+
+.summits-list {
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+}
+
+.summits-heading {
+  text-align: center;
+  color: #383b42;
+  font-size: 1.75rem;
+  position: relative;
+  margin-bottom: 1.5rem;
+}
+
+.summits-heading::after {
+  content: " ";
+  position: absolute;
+  bottom: -0.8rem;
+  left: 50%;
+  background: #ec5242;
+  width: 2rem;
+  height: 0.1rem;
+  transform: translateX(-50%);
+}
+
+.summits-text {
+  font-size: 1rem;
+  line-height: 1.8;
+  margin-bottom: 1.9rem;
+}
+
+.summits-list-item img {
+  object-fit: cover;
+  opacity: 0.7;
+  mix-blend-mode: multiply;
+  height: 100%;
+  width: 100%;
+}
+
+.summits-list-item {
+  position: relative;
+  background: #ec5242;
+}
+
+.summits-list-item__details {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 300px;
+  transform: translate(-50%, -50%);
+  color: #fff;
+}
+
+.summits-list-item__details h6 {
+  font-size: 1.6rem;
+}
+
+.summits-list-item__details p {
+  font-size: 1rem;
+}
+
+.summits-list-item:not(:last-child) {
+  margin-bottom: 1rem;
+}
+
+/* THIS IS THE START OF MORE INFO IN THE ABOUT US PAGE STYLING */
+
+.more-info {
+  background-image: url(images/background.svg);
+  color: #fff;
+  padding: 20px;
+}
+
+.more-info-detail {
+  font-size: 17px;
+  color: #272a31;
+  border: 1px solid #a9a9a9;
+  text-align: center;
+  line-height: 1.5rem;
+}
+
+.more-info h1 {
+  font-family: "cocogoose", sans-serif;
+  color: #ec5242;
+  text-align: center;
+  padding: 10px 0;
+}
+
+.email-us {
+  color: #272a31;
+  padding: 20px;
+  text-align: center;
+}
+
+/* STYLING FOR THE LOGO SECTION IN THE ABOUT PAGE */
+
+.prize-logo img {
+  border: 1px solid #d3d3d3;
+  width: 40%;
+  height: 50%;
+  padding: 20px;
+}
+
+.prize-text {
+  text-align: center;
+  line-height: 1.5rem;
+  margin: 20px;
+}
+
+/* THIS IS THE STYLING FOR THE PARTNERS SECTION */
+
+.partners {
+  background-color: #272a31;
+  height: 300px;
+}
+
+.partners-heading {
+  color: #fff;
+  position: relative;
+}
+
+.partners-heading::after {
+  content: " ";
+  position: absolute;
+  bottom: -1rem;
+  left: 50%;
+  background: #ec5242;
+  width: 2rem;
+  height: 0.1rem;
+  transform: translateX(-50%);
+}
+
+.partners-list {
+  padding-left: 0;
+}
+
+.partners-list img {
+  padding: 5px;
+  align-items: center;
+}
+
+/* FOOTER SECTION */
+
+footer {
+  display: flex;
+  justify-content: center;
+  margin: 10px;
+  gap: 100px;
+  background-color: #fff;
+  color: #272a31;
+}
+
+footer .rights {
+  padding-top: 5px;
+  line-height: 1.5rem;
+  font-size: 13px;
+}
+
+@media screen and (max-width: 600px) {
+  .topnav.responsive {
```

Comment:
- [ ] As the previous reviewer mentioned, the mobile nav should follow the colour of the design template. `Blue` is __not part__ of the colour of the design template. Kindly change it or return it to `default (plain black)`

hamburger | nav items
----- | -----
![image](https://user-images.githubusercontent.com/16058470/124268972-317c2780-db32-11eb-8efb-f456c4283574.png) | ![image](https://user-images.githubusercontent.com/16058470/124268927-1c06fd80-db32-11eb-8b70-af2d7fba9152.png)